### PR TITLE
docs(all): update README & jsdoc

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
#174, #188 

## Description
Check just before 3.0 release.

### CODE REVIEW

File: src/PanoViewer/PanoViewer.js [hunk 1]
```diff
@@ -378,70 +413,87 @@ export default class PanoViewer extends Component {
...
+	 * @return {eg.view360.PanoViewer} PanoViewer instance<ko>PanoViewer 인스턴스</ko>
 	 */
 	setUseZoom(useZoom) {
 		if (typeof useZoom !== "boolean") {
-			return;
+			return this;
 		}
 
 		this._yawPitchControl.option("useZoom", useZoom);
+		return this;
```
Comment (happyhj) [inline src/PanoViewer/PanoViewer.js]: You can make the line of code shorter by applying the following changes.
```js
typeof useZoom === "boolean" && this._yawPitchControl.option("useZoom", useZoom);
return this;
```

File: src/PanoViewer/PanoViewer.js [hunk 2]
```diff
@@ -160,10 +180,12 @@ export default class PanoViewer extends Component {
 	}
```
Comment (happyhj) [inline src/PanoViewer/PanoViewer.js]: You can make the line of code shorter by applying the following changes.
```js
if (video) {
    this.setImage( ... )
}
return this;
```

File: src/PanoViewer/PanoViewer.js [hunk 3]
```diff
@@ -217,11 +243,12 @@ export default class PanoViewer extends Component {
...
```
Comment (happyhj) [inline src/PanoViewer/PanoViewer.js]: You can make the line of code shorter by applying the following changes.
```js
	setImage(image, param = {}) {
		const cubemapConfig = Object.assign({
			order: "RLUDBF",
			tileConfig: {
				flipHirozontal: false,
				rotation: 0
			}
		}, param.cubemapConfig);
		const isVideo = !!(param.isVideo);

		if (this._image && this._isVideo !== isVideo) {
			/* eslint-disable no-console */
			console.warn("Currently not supporting to change content type(Image <--> Video)");
			/* eslint-enable no-console */
			return this;
		}

		if (image) {
            this._image = image;
            this._isVideo = isVideo;
            this._projectionType = param.projectionType || PROJECTION_TYPE.EQUIRECTANGULAR;
            this._cubemapConfig = cubemapConfig;

            this._deactivate();
            this._initRenderer(this._yaw, this._pitch, this._fov, this._projectionType, this._cubemapConfig);
		}

		return this;
	}
```

File: src/PanoViewer/PanoViewer.js (hunk 1 above)
Comment (jongmoon) [reply src/PanoViewer/PanoViewer.js]: @happyhj Ok, I'll take it

**Pull request merged**